daemon/libs/detour/DetourNode.h

```diff
@@ -46,7 +46,6 @@ class dtNodePool
 public:
 	dtNodePool(int maxNodes, int hashSize);
 	~dtNodePool();
-	inline void operator=(const dtNodePool&) {}
```

Comment:
Looks like libs in daemon/libs were external at some point. But i don't know in what state are these now. For some libs srcs are found, but look significantly different (our libs are outdated?).

For detour lib, i haven't found public src. 
If you can point me to correct one and we can update our detour lib version from public one, i suggest a patch to detour team.